modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1198,13 +1198,19 @@ Maximum number of bytes returned in a fetch request.
 
 The strategy used to fulfill fetch requests.
 
+* `polling`: 
```

Comment:
Why no description for this option?

Reply:
I'd mark it as deprecated, then give it a description like;

```
Repeatedly polls every partition in the request for new data. The polling interval is set by <<fetch_reads_debounce_timeout,`fetch_reads_debounce_timeout`>
```

Reply:
Added you as a reviewer @mattschumpert because this suggests we add this (`polling` option)to our list of deprecations. I know you mentioned a wile back that we should alert you on any that we add.